### PR TITLE
pagmo 2.16.0

### PR DESCRIPTION
**Pull request by alebcay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/pagmo.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Pagmo < Formula
   desc "Scientific library for massively parallel optimization"
   homepage "https://esa.github.io/pagmo2/"
-  url "https://github.com/esa/pagmo2/archive/v2.15.0.tar.gz"
-  sha256 "07977be690b512ea11fc40d5b8bfa0a7a8507ce9053a615c2bc4725d355ef9a8"
+  url "https://github.com/esa/pagmo2/archive/v2.16.0.tar.gz"
+  sha256 "076918ca975e2b45eedd85b65da0de650d4bd0b3f86182c0c144c7fdc191185b"
   license "GPL-3.0"
```
Comment (equal-l2) [inline Formula/pagmo.rb:6]:
```suggestion
  license "GPL-3.0-or-later"
```
(guessed from https://github.com/esa/pagmo2/blob/cc82c955f6027fba8a403550fc83c05d2f857afb/src/algorithm.cpp#L10)

**Comment by SMillerDev** [comment]
This seems to need a patch for the newest boost

**Comment by chenrui333** [comment]
license file update
```
/* Copyright 2017-2020 PaGMO development team

This file is part of the PaGMO library.

The PaGMO library is free software; you can redistribute it and/or modify
it under the terms of either:

  * the GNU Lesser General Public License as published by the Free
    Software Foundation; either version 3 of the License, or (at your
    option) any later version.

or

  * the GNU General Public License as published by the Free Software
    Foundation; either version 3 of the License, or (at your option) any
    later version.

or both in parallel, as here.

The PaGMO library is distributed in the hope that it will be useful, but
WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
for more details.

You should have received copies of the GNU General Public License and the
GNU Lesser General Public License along with the PaGMO library.  If not,
see https://www.gnu.org/licenses/. */
```

**Comment by chenrui333** [comment]
```
==> brew audit pagmo --online --git --skip-style
==> FAILED
Error: GitHub API Error: API rate limit exceeded for 207.254.26.156. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)
Try again in 3 minutes 2 seconds, or create a personal access token:
  https://github.com/settings/tokens/new?scopes=gist,public_repo&description=Homebrew
echo 'export HOMEBREW_GITHUB_API_TOKEN=your_token_here' >> ~/.zshrc
```

**Pull request closed without merging**